stagecoach/price-pusher/price_pusher/core/pusher.py

```diff
@@ -0,0 +1,33 @@
+from abc import ABC, abstractmethod
+
+from typing import List, Optional, Dict
+
+from pragma.publisher.client import PragmaClient
+from pragma.core.entry import Entry
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class IPricePusher(ABC):
+    @abstractmethod
+    async def update_price_feeds(self, entries: List[Entry]) -> Optional[Dict]: ...
+
+
+class PricePusher(IPricePusher):
+    def __init__(self, client: PragmaClient) -> None:
+        self.client = client
+
+    async def update_price_feeds(self, entries: List[Entry]) -> Optional[Dict]:
+        """
+        Push the entries passed as parameter with the internal pragma client.
+        """
+        logger.info(f"👷‍♂️ PUSHER processing {len(entries)} new assets to push...")
+        try:
+            response = await self.client.publish_entries(entries)
```

Comment:
not sure that always returns a dict tho

Reply:
do we have an idea of all expected responses or shall I put an Any?
Ideally we would describe all possibles responses like
```python
def update_price_feeds(self, entries: List[Entry]) -> Optional[PublishResponse]
```

Reply:
well we can define two types for the answer, the API response and the on-chain one